azext_hanaonazure/modules_sdk/operations/hana_instances_operations.py

```diff
@@ -237,3 +238,56 @@ def get(
 
         return deserialized
     get.metadata = {'url': '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HanaOnAzure/hanaInstances/{hanaInstanceName}'}
+
+    def restart(
+            self, resource_group_name, hana_instance_name, custom_headers=None, raw=False, **operation_config):
+        """The operation to restart a SAP HANA instance.
+
```

Comment:
Add a little bit more description here (get has an extended description of "Gets properties of a SAP HANA instance for the specified subscription, resource group, and instance name.")

Reply:
This is auto generated